packages/testing/src/api/directive.ts

```diff
@@ -32,6 +32,50 @@ interface DirectiveMocker {
   (directive: TransformerDirective, executionMock: TransformerMock): any
 }
 
+/**
+ *
+ * @description
+ * Used for writing directive tests for asynchronous directives e.g.
```

Comment:
Seemed a little redundant to say directive twice:
```suggestion
 * Used for writing tests for asynchronous directives e.g.
```